scripts/check-screenshots.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+FILES_DIFF=$(diff <(find docs/images -type f -name "screen-?.png" -exec md5sum {} + | sort -k 2 | sed 's/ .*\// /') <(find fastlane/metadata/android/en-US/images/phoneScreenshots -type f -exec md5sum {} + | sort -k 2 | sed 's/ .*\// /'))
+if [[ $FILES_DIFF ]]; then
+    echo -e "\033[0;31mScreenshots in docs/images and fastlane/metadata/android/en-US/images/phoneScreenshots are not same\033[0m" >&2
+    exit 5;
```

Comment:
Why not exit 1?

Reply:
Yes 1 is for general errors, we can use that as well. I just wanted to make it unique to our application so I chose a random 5

Reply:
I think it is better to use standard errors. CI may be hardcoded to understand only 1 exit code

Reply:
@iamareebjamal yes true, updated with `exit 1`